server/src/main/java/io/spine/examples/chatspn/server/chat/ChatMembersProjection.java

```diff
@@ -52,4 +53,9 @@ void on(GroupChatCreated e) {
                  .addMember(e.getCreator())
                  .addAllMember(e.getMemberList());
     }
+
+    @Subscribe
+    void on(MembersIncluded e) {
+        builder().addAllMember(e.getMemberList());
```

Comment:
This is dangerous. I would try to take some precautions to prevent duplicates, in any case.